src/index.html

```diff
@@ -2,11 +2,22 @@
 <html>
   <head>
     <title>Document Two Sections</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style.css" />
```

Comment:
No need to add `/` here. Can you remove it?
```suggestion
    <link rel="stylesheet" href="style.css">
```